caikit_nlp/modules/text_generation/text_generation_local.py

```diff
@@ -624,8 +623,8 @@ def tokenize_function_seq2seq(
     max_target_length: int,
 ):
     """Tokenization function to be used for seq2seq training; this function consumes a
-    GenerationTrainRecord object and applies the verbalizer to it followed by
-    the model tokenizer. Finally, we postprocess by ignoring pad tokens in the label IDs.
+    GenerationTrainRecord object and postprocesses by ignoring pad tokens in the
```

Comment:
Yup, I can work on that - if needed, I can get rid of the builder pattern or something like that to avoid passing a closure, which might be a little gross, but will at least not duplicate stuff